crates/pop-common/src/helpers.rs

```diff
@@ -40,6 +40,32 @@ pub fn get_project_name_from_path<'a>(path: &'a Path, default: &'a str) -> &'a s
 	path.file_name().and_then(|name| name.to_str()).unwrap_or(default)
 }
 
+/// Determines the target triple based on the current platform.
+pub fn target() -> Result<&'static str, Error> {
```

Comment:
Personal view.  I dont like `helpers` and `utils`, especially when the root module is empty, as it seems lazy and can potentially make stuff hard to find for users of the crate. From a crate consumer perspective, would it not be easier to find `target` within `pop_common`? From a scope perspective, does `target` not potentially apply to the whole crate in terms of usage. In other words, structuring things hierarchically from general to specific.

If we dont want it in the root, then would something like `platform` not be more descriptive/useful then `helpers`?

Only mentioning it as it starts out as ill-considered and then everybody else just follows the same approach and then we end up with a module with a whole lot of random functions that dont have any real connection.